src/Config/Loader/PhpFileLoader.php

```diff
@@ -32,6 +32,46 @@ public function load($file, $type = null)
     {
         $code = rtrim(file_get_contents($file));
 
+        // declare() statements
+        $codeNew = '';
+        $tokens = token_get_all($code);
+        $collect = false;
+        $declarePart = '';
+
+        foreach ($tokens as $k => $token) {
+            if (is_array($token)) {
+                if (T_DECLARE === $token[0]) {
+                    $collect = true;
+                }
+
+                if ($collect) {
+                    $declarePart .= $token[1];
+                } else {
+                    $codeNew .= $token[1];
+                }
+
+            } else {
+                if ($collect && ';' === $token) {
+                    $declarePart = preg_replace('/([\s]*,)?strict_types[\s]*=[\s]*1([\s]*,)?/', '', $declarePart);
```

Comment:
You're right. Added another test case and fixed in 1e0e9263f5cc64416058792c8d5a1ea7c1042b8b.